test/runtests.jl

```diff
@@ -293,4 +293,50 @@ end
     end
 end
 
+@testset "Create Atoms" begin
+    LMP(["-screen", "none"]) do lmp
+        command(lmp, """
+            atom_modify map yes
+            region cell block 0 2 0 2 0 2
+            create_box 1 cell
+            lattice sc 1
+        """)
+        x = rand(3, 100)
+        id = Int32.(collect(1:100))
+        types = ones(Int32, 100)
+        image = ones(Int32, 100)
+        v = rand(3, 100)
+
+
+        create_atoms(lmp, x, id, types, v=v, image=image, bexpand=true)
```

Comment:
```suggestion
        create_atoms(lmp, x, id, types; v, image, bexpand=true)
```

Reply:
Sorry about that. I'll be sure to pay more attention.